docs/index.rst

```diff
@@ -4,19 +4,21 @@ Hafnian
 :Release: |release|
 :Date: |today|
 
-The fastest exact hafnian library.
+The fastest hafnian library.
```

Comment:
Also the only hafnian library 😆 
I want to say 'Fastest Hafnian library in the North (FHLN)', but then it makes it sound like there is a faster one in the south...